ghcide/ghcide.cabal

```diff
@@ -1,32 +1,32 @@
-cabal-version:      3.4
-build-type:         Simple
-category:           Development
-name:               ghcide
-version:            2.11.0.0
-license:            Apache-2.0
-license-file:       LICENSE
-author:             Digital Asset and Ghcide contributors
-maintainer:         Ghcide contributors
-copyright:          Digital Asset and Ghcide contributors 2018-2020
-synopsis:           The core of an IDE
-description:        A library for building Haskell IDE's on top of the GHC API.
+cabal-version: 3.4
+build-type: Simple
+category: Development
+name: ghcide
+version: 2.11.0.0
+license: Apache-2.0
+license-file: LICENSE
+author: Digital Asset and Ghcide contributors
+maintainer: Ghcide contributors
+copyright: Digital Asset and Ghcide contributors 2018-2020
+synopsis: The core of an IDE
+description: A library for building Haskell IDE's on top of the GHC API.
 homepage:
   https://github.com/haskell/haskell-language-server/tree/master/ghcide#readme
 
-bug-reports:        https://github.com/haskell/haskell-language-server/issues
-tested-with:        GHC == {9.12.2, 9.10.1, 9.8.4, 9.6.7}
+bug-reports: https://github.com/haskell/haskell-language-server/issues
+tested-with: ghc ==9.12.2 || ==9.10.1 || ==9.8.4 || ==9.6.7
```

Comment:
Starting with version 1.5.0.3, Gild will correctly leave this in set-builder notation. See https://github.com/tfausak/cabal-gild/pull/118.